VideoCallKitQuickStart/ViewController+CallKit.swift

```diff
@@ -92,8 +92,6 @@ extension ViewController : CXProviderDelegate {
     func provider(_ provider: CXProvider, perform action: CXEndCallAction) {
         NSLog("provider:performEndCallAction:")
 
-        // AudioDevice is enabled by default
-        self.audioDevice.isEnabled = true
```

Comment:
Because of this the app was throwing following error logs. @bchen-twilio has PR for this in Voice QS - https://github.com/twilio/voice-quickstart-swift/pull/295
```
2020-01-15 16:53:46.745192-0800 VideoCallKitQuickStart[6502:2330920] [aurioc] AURemoteIO.cpp:1590:Start: AUIOClient_StartIO failed (561145187)
```